spec/requests/v6/assessments_controller_spec.rb

```diff
@@ -1749,6 +1749,36 @@ def cash_transactions(amount)
           end
         end
       end
+
+      context "redact response timestamp" do
+        context "with successful submission" do
+          let(:params) { {} }
+
+          it "returns http success" do
+            expect(response).to have_http_status(:success)
+          end
+
+          it "returns timestamp attribute in response" do
+            expect(parsed_response).to be_key(:timestamp)
+          end
+
+          it "redacts time in timestamp" do
```

Comment:
It would be nicer to control the timestamp in the test, so that the test doesn't mirror the code e.g. set date to 27th June 2023 and expect timestamp to be 2023-06-27